api/routes/userRoute.js

```diff
@@ -2,5 +2,6 @@ const router = require('express').Router();
 const controller = require('../controllers/user');
 
 router.post('/:username/profile', controller.createUserProfile);
+router.put('/:username/profile', controller.updateUserProfile);
```

Comment:
since you are updating some fields and not replacing the entire file, `patch` is the way to go for convention